source/library/init-player.js

```diff
@@ -0,0 +1,62 @@
+import {prefix} from './get-vendor-prefix';
+import ElementStub from './element-stub';
+
+/**
+ * Initialize a WebAnimationsPlayer instance
+ * @param  {HTMLElement} element   HTMLElement to instantiate on
+ * @param  {array} keyframes   keyframes passed to render
+ * @param  {object} options    options passed to render
+ * @param  {function} [render] render function used to apply interpolated inline styles
+ * @param window {Window} [window=global.window] Global context to use
+ * @param document {Document} [document=global.window] Document context to use
+ * @return {object}         WebAnimationsPlayer instance
+ */
+export default function initPlayer(element, keyframes, options, render, window = global.window, document = global.document) {
+	// Create an element stub for the web animation polyfill to write on if options.render is defined
+	const playerElement = render ? ElementStub.create(element, window) : element;
```

Comment:
`ElementStub` sounds much like testing and not something that is used in production code.
Maybe you could rename it to `StyleProxy` or something similar.